src/configure/model/models.ts

```diff
@@ -85,6 +85,7 @@ export interface PipelineTemplate {
     label: string;
```

Comment:
do you want to remove GitHubPAT as input from WizardInputs ?

Reply:
Will do that.